src/main/java/innovimax/mixthem/operation/DefaultLineAlternation.java

```diff
@@ -1,5 +1,6 @@
 package innovimax.mixthem.operation;
 
+import static innovimax.mixthem.MixConstants.*;
```

Comment:
please remove all import static. it's confusing

Reply:
ok